src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -57,7 +57,7 @@ void fx_muxer_t::display_missing_framework_error(
     trace::error(_X("  - Check application dependencies and target a framework version installed at:"));
     trace::error(_X("      %s"), fx_ver_dirs.c_str());
     trace::error(_X("  - Installing .NET Core prerequisites might help resolve this problem:"));
-    trace::error(_X("      %s"), DOTNET_CORE_GETTING_STARTED_URL);
+    trace::error(_X("      %s"), DOTNET_CORE_INSTALL_PREREQUISITES_URL);
```

Comment:
This seems to be noise.  Why are we providing prerequisite instructions here.  Should be on the download page if it is required, but really should just be taken care of by the installer.

Reply:
Imoved this to the end of the error message to show that it's least likely to help. But I think it doesn't hurt to have it here.